MEMBER_TRAVEL_FUND.md

```diff
@@ -6,12 +6,13 @@ and the Node.js Foundation projects.
 
 ## Restrictions
 
-* Candidates must be an individual member of the Node.js Foundation.
+* Candidates must be an individual member of the Node.js Foundation. The membership
+  status can be checked on [the foundation website][].
 
 ## Process
 
-Any member can apply for travel funds. Requests must be made prior to the start
-of travel. Requests submitted after travel has started will not be approved.
+Requests must be made prior to the start of travel.
```

Comment:
**Any member can apply for travel funds.**

Isn't this still valid? OR Is there a specific reason to remove this line?
I think for new members and people who are requesting fund for the first time, this line is very clear and welcoming as it should be.

Reply:
It basically just repeats the paragraph before that, and doesn't mention what "member" actually is - to explain about this we will need to repeat more.